apps/bas/src/App.tsx

```diff
@@ -1,15 +1,15 @@
 import React, { useState } from 'react'
 import {
+  AnyCardData,
   BallotStyle,
-  CardData,
-  Optional,
-  OptionalElection,
+  safeParseElection,
+  safeParseJSON,
+  schema,
   VoterCardData,
 } from '@votingworks/types'
+import { Card } from '@votingworks/utils'
 
-import fetchJSON from './utils/fetchJSON'
-
-import { CardAPI, EventTargetFunction } from './config/types'
+import { EventTargetFunction } from './config/types'
 
 import useStateAndLocalStorage from './hooks/useStateWithLocalStorage'
```

Comment:
I know this isn't really related to this PR but its a little confusing and seems unneccessary that this is renamed from useStateWithLocalStorage to useStateAndLocalStorage